### PR TITLE
ert fix + new ert species

### PR DESCRIPTION
**Pull request by robustinII**
## What Does This PR Do
This pr adds three new races for ERT to choose: Slime people, plasmamen and IPCs also adds logical names for ert members. No anymore Unathi Sergeant Smiths.

## Why It's Good For The Game
new rp (and not only!) situations
diversity.

## Images of changes
изображение_2024-04-13_144241709

## Testing
I tested almost everything I guess.

## Changelog
:cl:
edit: ert.dm
ert member appearance system change a bit, plus logical names for xenoraces

edit: ert_outfits.dm
some little additional stuff like tanks and suits for plasmaman plus changed a bit commander rank and age change system

edit: language.dm
kidan's and nian's languages proc get_random_name changed a bit.

add: survival boxes for ert plasmamen and ipcs

/:cl:

### CODE REVIEW

**Comment by robustinII** [comment]
wtf is conflict and how to resolve it and why this thing appears

**Comment by SynthTwo** [comment]
The naming scheme is a great idea, but there is reasons why those species got excluded.

**Comment by robustinII** [comment]
> The naming scheme is a great idea, but there is reasons why those species got excluded.

and whyyyyyy

**Comment by GDNgit** [comment]
> > The naming scheme is a great idea, but there is reasons why those species got excluded.
> 
> and whyyyyyy

Plasmamen sit on a 50% burn weakness (lmao should be removed at some point but talk to design before doing so). Slimes not having bones can cause some weirdness with combat, but this is mostly a nukie issue than an ert one. IPCs are probably fine? but the worry was that they'd walk into a situation, get merc'd, then wouldn't even be able to call for backup because of oldcrit.

**Comment by Coolrune206** [comment]
Please don't let ERTs be IPCs.
The last thing that is needed is to send in a Gamma-level lootbox that costs two EMPs to open.

File: code/modules/response_team/ert.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@ GLOBAL_LIST_EMPTY(ert_request_messages)
 		A.close()
 	for(var/mob/M in response_team_members)
 		ert_role_prefs.Add(input_ranked_async(M, "Please order ERT roles from most to least preferred (20 seconds):", GLOB.active_team.get_slot_list()))
-	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(dispatch_response_team), response_team_members, ert_gender_prefs, ert_species_prefs, ert_role_prefs), 20 SECONDS)
+	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(dispatch_response_team), response_team_members, ert_gender_prefs, ert_species_prefs, ert_role_prefs), 10 SECONDS)
```
Comment (lewcc) [inline code/modules/response_team/ert.dm:110]: Undocumented change?

Comment (robustinII) [reply code/modules/response_team/ert.dm:110]: I forgot about this

**Comment by robustinII** [comment]
> Please don't let ERTs be IPCs. The last thing that is needed is to send in a Gamma-level lootbox that costs two EMPs to open.

thanks for idea - HARDENED IPCS

**Comment by GDNgit** [comment]
> > Please don't let ERTs be IPCs. The last thing that is needed is to send in a Gamma-level lootbox that costs two EMPs to open.
> 
> thanks for idea - HARDENED (emp proof) IPCS

I'd keep this out of the original pr and just lower the scope, maybe just add plasmamen and slime people and keep the other stuff out of it. Qwerty is also going to rework plasmamen coming up to not have the 50% hell weakness

**Comment by robustinII** [comment]
why I can't just revert 803 changes? that's not even my stuff! not cool!

**Comment by robustinII** [comment]
Okay, I have no idea what is happening, I have 100 errors, station map files are deleted and other weird stuff. This PR and branch are cursed.

**Pull request closed without merging**